tests/test_validators_dataclass.py

```diff
@@ -41,9 +43,7 @@ def check_a2(cls, v: List[int]) -> List[int]:
     assert MyDataclass(a=[1, 2]).a == [1, 2, 123, 456]
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_validate_multiple():
-    # also test TypeError
```

Comment:
Added a test below demonstrating that type errors aren't handled by the validators